packages/amplify-console-integration/projects-templates/create-react-app-auth-amplify/CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,5 @@
 ## Code of Conduct
+
```

Comment:
why?

Reply:
why the added empty line? 
it was not manually added, but introduced by `npm run commit`

Reply:
removed the file